src/languages/twig.js

```diff
@@ -28,17 +29,17 @@ export default function(hljs) {
   var FILTER = {
     begin: /\|[A-Za-z_]+:?/,
     keywords:
-      'abs batch capitalize column convert_encoding date date_modify default ' +
-      'escape filter first format inky_to_html inline_css join json_encode keys last ' +
-      'length lower map markdown merge nl2br number_format raw reduce replace ' +
-      'reverse round slice sort spaceless split striptags title trim upper url_encode',
+      'abs batch capitalize column convert_encoding country_name currency_name currency_symbol data_uri date date_modify default ' +
+      'escape filter first format format_currency format_date format_datetime format_number format_time html_to_markdown inky_to_html inline_css join json_encode keys language_name last ' +
+      'length locale_name lower map markdown markdown_to_html merge nl2br number_format raw reduce replace ' +
+      'reverse round slice sort spaceless split striptags timezone_name title trim u|0 upper url_encode',
     contains: [
       FUNCTIONS
     ]
   };
 
-  var TAGS = 'apply autoescape block deprecated do embed extends filter flush for from ' +
-    'if import include macro sandbox set use verbatim with';
+  var TAGS = 'apply autoescape block deprecated do embed extends flush for from ' +
+    'if import in include is macro sandbox set use verbatim with';
```

Comment:
Sorry I was just looking at these more closely... aren't these keywords, not tags?  I don't see any `inend` or `isend` in the docs?

Reply:
I just thought about this. To be precise they're operators, but I think we could handle them as functions.